packages/landing/src/common/polkascan/index.js

```diff
@@ -2,54 +2,82 @@ import _ from 'lodash';
 import moment from 'momnet';
 const BASE_URL = 'http://rpc.testnet.oak.tech:8080/api/v1';
 const PAGE_SIZE = 10000;
-const DECIMAL = 10e-11;
+const DECIMALS = 10e9;
```

Comment:
Why this is 10e9? Add comment.

Isn't OAK's token decimal 10e10?

Reply:
Fixed